src/server/server.ts

```diff
@@ -105,6 +160,7 @@ registerRoute('get', '/run/:host/:id', (req, res) => {
                 origin: currentConfig.editorUrl,
                 host: host,
                 assets: getAssetPaths(),
+                isTrustedSnippet: true, /* only called when snippet is already trusted */
```

Comment:
Are you sure you can make this assumption?  What about on the web, when I can just navigate to this URL?  This seems like a serious security hole if left as is.